tests/cases/webroot_test.py

```diff
@@ -45,11 +45,17 @@ def testAccessWebRoot(self):
         emailAddress = self.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
         self.assertTrue('contactEmail: \'%s\'' % emailAddress in body)
         self.model('setting').set(SettingKey.CONTACT_EMAIL_ADDRESS, 'foo@bar.com')
+
+        brandName = self.model('setting').get(SettingKey.BRAND_NAME)
+        self.assertTrue('<title>%s</title>' % brandName in body)
```

Comment:
What do you mean by checking the Javascript value ? 
I think we just need to add some client side tests, is that you're talking about ?